src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -458,6 +483,13 @@ subroutine mixedlayer_restrat_OM4(h, uhtr, vhtr, tv, forces, dt, h_MLD, VarMix,
     u_star = max(CS%ustar_min, 0.5*(U_star_2d(i,j) + U_star_2d(i+1,j)))
 
     absf = 0.5*(abs(G%CoriolisBu(I,J-1)) + abs(G%CoriolisBu(I,J)))
+    ! Compute I_LFront = 1 / (frontal length scale) [m-1]
+    lfront = 0.5 * (mle_fl_2d(i,j) + mle_fl_2d(i+1,j))
```

Comment:
I think mle_fl_2d(i,j) is either out-of-bounds or uninitialized when i=is-1. This is probably contributing to the dimensional scaling failurem